### DISCUSSION TITLE
Managed Package Availability

### DISCUSSION THREAD

**Discussion by Johno72** [jamessimone/apex-rollup#402] [Q&A]
Hi James,

I'm curious if there's an Apex Rollup Managed Package available on the AppExchange? 

Thanks in advance.

Cheers,
John

**Comment by jamessimone** [comment]
There is not (yet), but it’s on my radar as an option to introduce this year. No firm timeline at the moment, I’m afraid.

**Reply by Johno72** [reply]
Thanks James, I look forward to hearing more later in the year. 

Some thoughts on the benefits

It would be great to have a managed package knowing it has been though the security review by Salesforce, this makes it an easier conversation with management and interal governance admins may encounter (I know its open source, but there are lots of folks that don't have the skills to be 100% comfortable). It would be great receiving updates when available without upgrading one version to another. Salesforce consulting partners would be more likely to recommend or include Apex Rollup as the default solution. Anyway, thanks for your reply and excited to hear its on your radar.

Cheers,
John

**Reply by jamessimone** [reply]
@Johno72 totally understood, and you're not the first to make these points. I had already talked with @jongpie about going through the security review process for Nebula Logger to get an idea of what that same process might look like for Apex Rollup. I'll keep you updated!

**Reply by Johno72** [reply]
Thanks James, appreciate the consideration.
